app/Jobs/ValidateCSV.php

```diff
@@ -147,4 +157,22 @@ private function checkValueErrors($mismatch, WikidataValue $valueRule): ?string
 
         return null;
     }
+
+    private function checkMetaValueErrors($mismatch, MetaWikidataValue  $metaRule): ?string
+    {
+        $validator = Validator::make([
+            'meta_wikidata_value' => [
+                'property' => $mismatch['property_id'],
+                'meta_value' => $mismatch['meta_wikidata_value']
```

Comment:
Can we call the array key here `'meta_wikidata_value'` as well? (Inside the validator, the variable is also called `$metaWikidataValue` again.)